test/integration_test.go

```diff
@@ -4980,9 +4980,14 @@ func (ts *IntegrationTestSuite) TestScheduleUpdate() {
 		err = handle.Delete(ctx)
 		ts.NoError(err)
 	}()
+
+	stringKey := temporal.NewSearchAttributeKeyString("CustomStringField")
+	sa := temporal.NewSearchAttributes(stringKey.ValueSet("CustomStringFieldValue"))
+
 	updateFunc := func(input client.ScheduleUpdateInput) (*client.ScheduleUpdate, error) {
```

Comment:
May also be worth a test that just mutates a single search attribute on the existing collection and leaves the rest alone, but no big deal

Reply:
Test added. To be clear, the server behavior today is upsert. If we add a single search attribute on the existing collection, the entire collection is replaced with the single search attribute. This is inline with the CLI proposal as well:

For each `--schedule-search-attribute flag`, do a replace of the whole collection, including removing ones that were present before and not given.